src/components/varsom-avalanche-activity-obs/varsom-avalanche-activity-obs.tsx

```diff
@@ -10,7 +11,7 @@ import { Attachment } from '../varsom-observation/observation-model';
 })
 export class VarsomAvalancheActivityObs {
 
-  @Prop() strings: any;
+  @Prop({mutable: true}) strings: any;
```

Comment:
Denne trenger vel ikke sendes inn lengre?
Sette til private også?
Eller hvis du velger å flytte lasting av tekster til render-metoden, trenger du vel ikke denne klasse-variabelen i det hele tatt?

Reply:
Vi trenger ikke prop(), men det fungerte ikke å flytte det til render, siden rendermetoden ikke kan være async...
Vi trenger ikke sende inn parameter til getLocaleComponentStrings() i det hele tatt. Den kan kalle på getLocaleFromDom()

så det beste jeg får til er:

private strings: any;

async componentWillLoad(){
    this.strings = await getLocaleComponentStrings();
  }

Reply:
Ok!